### PR TITLE
Modify the url of the swagger-codegen-cli.jar

### PR DESCRIPTION
**Pull request by michalyao**
The jar which I download from the current url can't run the command properly.

### CODE REVIEW

**Comment by wing328** [comment]
@michalyao thanks for the PR. What about using this one instead? 

https://mvnrepository.com/artifact/io.swagger/swagger-codegen-cli/2.2.1

**Comment by michalyao** [comment]
Thanks for the suggestions. I fixed it :-)

File: README.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate \
 ```
 (if you're on Windows, replace the last command with `java -jar modules\swagger-codegen-cli\target\swagger-codegen-cli.jar generate -i http://petstore.swagger.io/v2/swagger.json -l php -o c:\temp\php_api_client`)
 
-You can also download the JAR (latest relesae) directly from [maven.org]( http://central.maven.org/maven2/io/swagger/swagger-codegen/2.2.0/swagger-codegen-2.2.0.jar)
+You can also download the JAR (latest relesae) directly from [maven.org](https://mvnrepository.com/artifact/io.swagger/swagger-codegen-cli/2.2.1)
```
Comment (wing328) [inline README.md]: please replace the URL with http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.2.1/swagger-codegen-cli-2.2.1.jar instead.

**Comment by wing328** [comment]
PR merged. Thanks for the contribution.

**Pull request merged**